lib/new_relic/agent/instrumentation/ruby_openai/instrumentation.rb

```diff
@@ -147,6 +148,22 @@ def update_chat_completion_messages(messages, response, summary)
       end
     end
 
+    def record_content_enabled?
+      NewRelic::Agent.config[:'ai_monitoring.record_content.enabled']
+    end
+
+    def remove_content(messages)
```

Comment:
I think these would be better implemented as methods that destructively operate on their input and don't return anything.

Reply:
Methods are now just destroying, no return